bundles/org.openhab.binding.metofficedatahub/src/main/resources/OH-INF/thing/channel-types.xml

```diff
@@ -0,0 +1,648 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="metofficedatahub"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:thing="https://openhab.org/schemas/thing-description/v1.0.0"
+	xsi:schemaLocation="https://openhab.org/schemas/thing-description/v1.0.0 https://openhab.org/schemas/thing-description-1.0.0.xsd">
+
+	<!-- Channel Groups for MetOfficeDataHub Binding - Site API -->
+	<channel-group-type id="site-common-data-grp">
+		<label>Common Site Data</label>
+		<description>Site Specific Location weather forecast - Common Data</description>
+		<channels>
+			<channel id="loc-name" typeId="loc-name-type"/>
+		</channels>
+	</channel-group-type>
+
+	<channel-group-type id="site-hr-forecast-grp">
+		<label>Hourly Forecast</label>
+		<description>Site Specific Location weather forecast - Hourly basis</description>
+		<channels>
+			<channel id="forecast-ts" typeId="time-type"/>
+			<channel id="air-temp-current" typeId="air-temp-current-type"/>
+			<channel id="air-temp-min" typeId="air-temp-min-type"/>
+			<channel id="air-temp-max" typeId="air-temp-max-type"/>
+			<channel id="feels-like" typeId="feels-like-type"/>
+			<channel id="humidity" typeId="humidity-type"/>
+			<channel id="visibility" typeId="visibility-type"/>
+			<channel id="precip-prob" typeId="precip-prob-type"/>
+			<channel id="precip-rate" typeId="precip-rate-type"/>
+			<channel id="precip-total" typeId="precip-total-type"/>
+			<channel id="snow-total" typeId="snow-total-type"/>
+			<channel id="uv-index" typeId="uv-index-type"/>
+			<channel id="pressure" typeId="pressure-type"/>
+			<channel id="wind-speed" typeId="wind-speed-type"/>
+			<channel id="wind-speed-gust" typeId="wind-speed-gust-type"/>
+			<channel id="wind-gust-max" typeId="wind-gust-max-type"/>
+			<channel id="wind-direction" typeId="wind-direction-type"/>
+			<channel id="dewpoint" typeId="dewpoint-type"/>
+		</channels>
+	</channel-group-type>
+
+	<channel-group-type id="site-hr-forecast-current-grp">
```

Comment:
This seems to be a duplicate of `site-hr-forecast-grp` either one can be removed.

Reply:
Thank you - silly mistake from my first time using groups :)